gcore/resource_gcore_subnet.go

```diff
@@ -260,7 +265,11 @@ func resourceSubnetCreate(ctx context.Context, d *schema.ResourceData, m interfa
 	}
 
 	log.Printf("Create subnet ops: %+v", createOpts)
-	results, err := subnets.Create(client, createOpts).Extract()
+	rc := GetConflictRetryConfig(SubnetResourceTimeoutMinutes)
```

Comment:
no-no-no, client should be able to specify them in terraform code

Reply:
use d.Timeout()

Reply:
to use specified by client for this resource value or default provided value, e.g.
```
d.Timeout(schema.TimeoutCreate).Seconds()
```